src/page/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,133 @@
+import { useEffect, useState } from 'react';
+import { useParams, NavLink } from 'react-router-dom';
+import cn from 'classnames';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage = () => {
+  const { slug } = useParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isEmpty, setIsEmpty] = useState<boolean>(false);
```

Comment:
you don't need a state for this
just create a common variable to store this - `!people.length`

general rule: no need to create a state variable if you can get its value from other state variables or props